### PR TITLE
✨ Add templates for creator follow for Writing NFT 

### PR DESCRIPTION
**Pull request by nwingt**
image

image

### CODE REVIEW

File: src/i18n/translations/en.json [hunk 1]
```diff
@@ -1,95 +1,85 @@
...
+  "transaction.subtitle": "You have received {amount} LikeCoin",
+  "unit.civic-likers": "{value, plural, one {Civic Liker} other {Civic Likers}}",
+  "unit.likers": "{value, plural, one {Liker} other {Likers}}",
+  "unit.likes": "{value, plural, one {Like} other {Likes}}",
+  "unit.works": "{value, plural, one {Work} other {Works}}",
+  "unsubscribe.text": "Unsubscribe",
+  "writing-nft_creator-follow_confirmation_content": "Please <a>click here</a> or the link below to confirm following {displayName}'s Writing NFT.",
+  "writing-nft_creator-follow_confirmation_subject": "Follow {displayName}'s Writing NFT",
+  "writing-nft_creator-follow_confirmation_title": "Follow {displayName}'s Writing NFT",
+  "writing-nft_creator-follow_publish-new_content": "<a>NFT_TITLE</a> is now live!",
```
Comment (williamchong) [inline src/i18n/translations/en.json:82]: `NFT_TITLE` is just a placeholder, it can be any value right?

File: src/i18n/translations/en.json [hunk 2]
```diff
@@ -1,95 +1,85 @@
...
+  "transaction.remarks": "Remarks： {remarks}",
+  "transaction.subject": "You have received {amount} LikeCoin",
+  "transaction.subtitle": "You have received {amount} LikeCoin",
+  "unit.civic-likers": "{value, plural, one {Civic Liker} other {Civic Likers}}",
+  "unit.likers": "{value, plural, one {Liker} other {Likers}}",
+  "unit.likes": "{value, plural, one {Like} other {Likes}}",
+  "unit.works": "{value, plural, one {Work} other {Works}}",
+  "unsubscribe.text": "Unsubscribe",
+  "writing-nft_creator-follow_confirmation_content": "Please <a>click here</a> or the link below to confirm following {displayName}'s Writing NFT.",
+  "writing-nft_creator-follow_confirmation_subject": "Follow {displayName}'s Writing NFT",
```
Comment (williamchong) [inline src/i18n/translations/en.json]: Any reason why only english is not in the format of `Writing NFT - Follow {displayName}` ?

**Review by williamchong — APPROVED** [review]

File: src/i18n/translations/en.json (hunk 1 above)
Comment (nwingt) [reply src/i18n/translations/en.json:82]: Yes

File: src/i18n/translations/en.json (hunk 2 above)
Comment (nwingt) [reply src/i18n/translations/en.json]: Not sure why, but seems like `Writing NFT - Follow {displayName}` is better for consistency

Comment (nwingt) [reply src/i18n/translations/en.json]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**